Add `stevedore`

kotx opened the pull request:
It's a bit janky to un/register services in Scoop manifests, so I hope I've done this correctly. 

Closes #295.

slonopotamus commented on bucket/stevedore.json:
```diff
@@ -0,0 +1,44 @@
+    },
+    "pre_install": "(Get-Service stevedored) -and (sudo Stop-Service stevedored) -and (sudo Remove-Service stevedored)",
+    "post_install": [
+        "(Get-LocalGroup -Name docker-users) -or (New-LocalGroup -Name docker-users -Description 'Users of Docker Desktop')",
```
You also want to add current user to `docker-users` group. Otherwise, user won't be able to access docker service.

slonopotamus commented on bucket/stevedore.json:
```diff
@@ -0,0 +1,44 @@
+    "homepage": "https://github.com/slonopotamus/stevedore",
+    "license": "Apache-2.0",
+    "notes": [
+        "To install Hyper-V: `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All -NoRestart`",
```
You possibly want to say that this needs to be done from elevated PowerShell.

slonopotamus commented on bucket/stevedore.json:
```diff
@@ -0,0 +1,44 @@
+{
```
Overall, I am a bit worried that this script duplicates lots of Stevedore installer logic. And things are guaranteed to diverge with later changes to Stevedore. Is it a common Scoop approach or it is possible to reuse more bits from upstream MSI?

kotx replied:
Actually, I realized I can use `msiexec` and use `TARGETDIR` to specify a directory, which is sufficient for nonportable. Would it be better to run with a dialog box/progress bar or completely unattended?

kotx commented:
I fixed the app name to be `stevedore-np` and used the same method of installation as most of the apps in this repository (`msiexec /qn`) and added a note for starting the `stevedored` service.

kotx commented:
Oh huh why is GitHub not showing the new commit

kotx commented:
It's been a few days- @TheRandomLabs could you take a look at this?

sitiom commented:
https://github.com/ScoopInstaller/Scoop/issues/4807 would have to be sorted out first.
